docs/configuration/service/ipoe-server.md

```
@@ -1,512 +1,1152 @@
+
+### RADIUS attributes
+
+When included in a RADIUS reply, the attributes below determine the
+client's IP address and prefix assignment, taking precedence over the
+corresponding local settings. If an attribute is omitted, the local
+configuration applies.
+
+The following table outlines the allocation behaviors for different
+RADIUS attributes:
+% stop_vyoslinter
+| RADIUS attribute | Allocation behavior with the RADIUS attribute | Allocation behavior without the RADIUS attribute |
```

markdownlint-cli2 (0.23.2)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)